src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx

```diff
@@ -0,0 +1,95 @@
+import Reflux from 'reflux';
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+import ApiMixin from '../../mixins/apiMixin';
+import GuideStore from '../../stores/guideStore';
+import GuideActions from '../../actions/guideActions';
+
+// GuideDrawer is what slides up when the user clicks on a guide cue.
+const GuideDrawer = createReactClass({
+  displayName: 'GuideDrawer',
+
+  propTypes: {
+    guide: PropTypes.object.isRequired,
+    closeHandler: PropTypes.func.isRequired,
+  },
+
+  mixins: [ApiMixin, Reflux.listenTo(GuideStore, 'onGuideStateChange')],
+
+  getInitialState() {
+    return {
+      step: 0,
+    };
+  },
+
+  onGuideStateChange(data) {
+    this.setState({
+      step: data.currentStep,
+    });
+  },
+
+  nextHandler() {
+    GuideActions.nextStep();
+  },
+
+  usefulHandler(useful) {
+    this.api.request('/assistant/', {
+      method: 'PUT',
+      data: {
+        guide_id: this.props.guide.id,
+        status: 'viewed',
+        useful,
+      },
+    });
+    this.props.closeHandler();
+  },
+
+  dismissHandler() {
+    this.api.request('/assistant/', {
+      method: 'PUT',
+      data: {
+        guide_id: this.props.guide.id,
+        status: 'dismissed',
+      },
+    });
+    this.props.closeHandler();
+  },
+
+  render() {
+    return (
+      <div>
+        <div className="assistant-drawer-title">
+          {this.props.guide.steps[this.state.step].title}
+        </div>
+        <div className="assistant-drawer-message">
+          {this.props.guide.steps[this.state.step].message}
+        </div>
+        <div>
+          {this.state.step < this.props.guide.steps.length - 1 ? (
+            <div>
+              <a className="btn btn-default" onClick={this.nextHandler}>
```

Comment:
Can you use the `Button` component?

Reply:
Yeah probably, did this because design elements are not final yet

Reply:
`Button` component has default styles you can use and extend to something more customized later when you're at design stage